app/views/projects/edit.html.erb

```diff
@@ -1,47 +1,45 @@
-<div class="p-4">
-  <div class="grid gap-4 xl:grid-cols-2 2xl:grid-cols-3">
-    <!--Edit general details form -->
-    <div
-      class="
-        p-4
-        bg-white
-        border
-        border-gray-200
-        rounded-lg
-        shadow-sm
-        2xl:col-span-2
-        dark:border-gray-700
-        sm:p-6
-        dark:bg-gray-800
-      "
-      data-controller="slugify"
-    >
-      <h2
-        class="
-          text-xl
-          font-semibold
-          leading-tight
-          text-gray-900
-          dark:text-gray-100
-        "
-      >
-        <%= t(:".general.title") %>
-      </h2>
-      <%= form_with(model: @project, url: namespace_project_path, method: :patch) do |form| %>
-        <div class="grid gap-4">
+<div class="p-4 md:mx-auto">
+  <div class="grid gap-4">
+
+    <h2 class="text-2xl font-extrabold dark:text-white">
+      <%= t(".general.title") %>
+    </h2>
+    <%= viral_card do |card| %>
+      <% card.header(title: t(".general.heading")) %>
+      <% card.section do %>
+        <%= form_with(model: @project, url: namespace_project_path, method: :patch, class: "grid gap-4") do |form| %>
           <%= form.fields_for :namespace, include_id: false do |builder| %>
-            <%= render partial: "projects/project_namespace_fields",
-            locals: {
-              builder: builder,
-              project: @project,
-              authorized_namespaces: @authorized_namespaces
-            } %>
+
+            <div class="form-field">
+              <%= builder.label :name %>
+              <%= builder.text_field :name,
+                                     required: true,
+                                     autofocus: true,
+                                     placeholder: t(:"projects.new.placeholder") %>
+            </div>
+            <div class="form-field">
+              <%= builder.label :description %>
+              <%= builder.text_area :description,
+                                    {
+                                      :class => "form-control",
+                                      "aria-label" => "Project description, not required"
```

Comment:
**nitpick (blocking):** Could you add a translation here please

Reply:
Updated in b6f1d20